index.js

```diff
@@ -50,6 +50,12 @@ module.exports = function inspect_ (obj, opts, depth, seen) {
         return inspect_(value, opts, depth + 1, seen);
     }
 
+    function indentedJoin(xs) {
```

Comment:
it seems like this function should take `opts.indent` as an argument, so it doesn't need to be recreated every time?